zeppelin-web/src/app/notebook/notebook.css

```diff
@@ -221,3 +232,41 @@
   text-decoration: none;
   cursor: default;
 }
+
+.userlist {
+  width: 320px;
+  font-family: Georgia, Times, serif;
+  font-size: 15px;
+  position: absolute;
+  z-index: 9999;
+}
+
+.userlist ul {
+  list-style: none;
+}
+
+.userlist ul li {
+  display: list-item;
+  text-decoration: none;
+  color: #000000;
+  background-color: #FFFFFF;
+  line-height: 30px;
+  border-bottom-style: none;
+  border-bottom-width: 1px;
+  border-bottom-color: #CCCCCC;
```

Comment:
It seems this line doesn't work. You can change this one to `border-bottom: 1px #CCCCCC solid;` then it will work :)